assets/general/bitcoin/info.json

```diff
@@ -18,11 +18,13 @@
   "createCoin": true,
   "consensus": "PoW",
   "blockTimeAvg": 600000,
-  "newPendingTxFetchInterval": 10000,
-  "oldPendingTxFetchInterval": 3000,
-  "registeredTxFetchInterval": 40000,
-  "newPendingTxFetchAttempts": 20,
-  "oldPendingTxFetchAttempts": 3,
+  "txFetchInfo": {
+    "newPendingTxFetchInterval": 10000,
```

Comment:
remove `TxFetch` from inner property names to make them shorter which was main idea of grouping